client/src/components/Tickets/TicketsLayout.scss

```diff
@@ -12,39 +12,33 @@
   .tickets-view {
     /* number */
     tr > *:nth-child(1) {
-      width: 7%;
-    }
-    /* subject */
-    tr > *:nth-child(2) {
-      width: 70%;
```

Comment:
**FP-1630**: I think these lines and [35-39](https://github.com/TACC/Core-Portal/pull/630/files#diff-17f912241e09b319c0126ba6791a8be4b1d36dc45bf3e19d12e2394c8ba7af3aL35-L39) explain the following bug.

| Loading Dashboard on This Branch | Loading Dashboard on `main`
| - | -
| ![FP-1630 - Branch - Table Too Short While Loading](https://user-images.githubusercontent.com/62723358/168382026-07f6a3e0-39f2-45b0-9955-b6292a3747b0.png) | ![FP-1630 - Main - Table Full Width While Loading](https://user-images.githubusercontent.com/62723358/168381779-1670b9c5-89f1-40d7-86eb-c05050828767.png)
| second column **not** visible[^1] | second column **is** visible

[^1]: I think the column has no width. And there is not content with which it can calculate leftover space to give it a width.

Reply:
@wesleyboar Resolved